genai-perf/genai_perf/config/generate/objective_parameter.py

```diff
@@ -37,3 +37,14 @@ class ObjectiveParameter:
     usage: SearchUsage
     category: ObjectiveCategory
     value: Any
+
+    def get_value_based_on_category(self) -> Any:
+        if (
+            self.category == ObjectiveCategory.INTEGER
+            or self.category == ObjectiveCategory.STR
+        ):
+            return self.value
+        elif self.category == ObjectiveCategory.EXPONENTIAL:
+            return 2**self.value
+
+        assert True, f"{self.category} is not a known ObjectiveCategory"
```

Comment:
Feels like this could be a bit more idiomatic if this was:

```python
raise ValueError(f"{self.category} is not a known ObjectiveCategory"
```

instead

Reply:
Good suggestion!